docs/viewpagerandroid.md

```diff
@@ -41,22 +41,22 @@ var styles = {
 
 ### Props
 
-* [View props...](view.md#props)
-
-- [`initialPage`](viewpagerandroid.md#initialpage)
-- [`keyboardDismissMode`](viewpagerandroid.md#keyboarddismissmode)
-- [`onPageScroll`](viewpagerandroid.md#onpagescroll)
-- [`onPageScrollStateChanged`](viewpagerandroid.md#onpagescrollstatechanged)
-- [`onPageSelected`](viewpagerandroid.md#onpageselected)
-- [`pageMargin`](viewpagerandroid.md#pagemargin)
-- [`peekEnabled`](viewpagerandroid.md#peekenabled)
-- [`scrollEnabled`](viewpagerandroid.md#scrollenabled)
-- [`setPage`](viewpagerandroid.md#setpage)
-- [`setPageWithoutAnimation`](viewpagerandroid.md#setpagewithoutanimation)
+- [View props...](view.md#props)
+
+* [`initialPage`](viewpagerandroid.md#initialpage)
```

Comment:
Are these changes are intentional?

Reply:
These changes are not intentional. They are done by the Prettier formatter if i am not wrong.

Reply:
I have corrected them to ```-``` instead of ```*``` on dda9272ce4c7a2b8f50e2e324d08ed13ca28086e to make them consistent.